CONTRIBUTING.md

```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## Rspec issues labels definition
+
+### `Your first PR` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
```

Comment:
`get started on` → `get started`

Reply:
`get started on` is the correct english here, an alternative would be `get started with`.